example/modelDataAndVideos.js

```diff
@@ -0,0 +1,34 @@
+const { PornHub } = require('../dist');
+
+(async () => {
+    let pornHub = new PornHub()
+    const lunaOkko = await pornHub.model('https://www.pornhub.com/model/luna-okko')
+
+    console.log(lunaOkko.name)
+    console.log(lunaOkko.about)
+    console.log(lunaOkko.bio)
+    for (const video of lunaOkko.mostRecentVideos) {
+        console.log(`${video.id} ${video.title} ${video.url}`)
+    }
+
+    pornHub = new PornHub()
+    const videoPage = await pornHub.video(lunaOkko.mostRecentVideos[0].url)
+    console.log(videoPage.uploadDate)
+
+    let hasMore = false
+    let pageCounter = 1
+    do {
+        pornHub = new PornHub()
```

Comment:
You don't need to re-initiate the instance.

Reply:
Yesterday I didn't, I've just added those. But when I tried it today I've got an error saying `HttpStatusError: 403 Forbidden at https://...` on the second request, but I saw that tests were working fine - but all of them are creating their own instance. So I added these lines to fix it. Any ideas what could be the problem? Am I on a blacklist or something?

Reply:
Oh really? Hmm just got report in #74 about 403. ok I will check them.